wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -602,6 +603,12 @@ instance BuildableSafeGen PublicKeyAsBase58 where
         %" }")
         ewalPublicKeyAsBase58
 
+instance FromHttpApiData PublicKeyAsBase58 where
+    parseQueryParam = Right . PublicKeyAsBase58Unsafe
```

Comment:
We don't have any smart constructor for this :thinking: ? So any piece of `Text` can be a `PublicKeyAsBase58` ?